testdata/command_spec.yml

```diff
@@ -0,0 +1,34 @@
+## Namespace for the command
+namespace: foo
+# Command name itself
+name: bar
+# Description of the command and what it does
+description: |
+  Lorem ipsum dolor sit amet.
+# Major and Minor version number (patch is automatic)
+version: 1.0
+# Format the command is in (see below for examples)
+# Valid options: habitat, docker, binary
+format: habitat
+
+# Habitat specific config
+# if format: habitat
+habitat:
+    mode: remote
+    package: core/git/2.14.1
+    # If local
+    # mode: local
+    # package: ./foobar.hart
+    command: git
+
+# Docker specific config
```

Comment:
This yaml example is looks invalid (there are three formats). We should use one format with one file.